src/api/FactsController.js

```diff
@@ -0,0 +1,14 @@
+export async function retrieveAnimalFact() {
+  const animalTypes = ['dog', 'cat'];
+  const randomAnimalType = randomIntFromInterval(0, animalTypes.length - 1);
+  const animalType = animalTypes[randomAnimalType];
+  const FACT_URL = `https://cat-fact.herokuapp.com/facts/random?animal_type=${animalType}&amount=1`;
+
+  const response = await fetch(FACT_URL);
+  const json = await response.json();
+  return json.text;
+}
+
+function randomIntFromInterval(min, max) {
```

Comment:
if function represents an action it should start with a verb (eg. getRandomIntFromInterval)